README.md

```diff
@@ -148,6 +148,7 @@ A curated list of awesome Java frameworks, libraries and software.
 
 *Frameworks and libraries that simplify reading/writing CSV data.*
 
+* [opencsv](http://opencsv.sourceforge.net) - Opencsv is a very simple csv parser library with a commercial-friendly license.
```

Comment:
Please condense the description to: Simple CSV parser with a commercial-friendly license.

Reason:

[Opencsv is a very]: It is implicite that you describe Opencsv, so don't be redundant.
[library]: Also implicit.